### PR TITLE
TD-1590/0-reviewers

### PR DESCRIPTION
**Pull request by ermirizio**
Update/Sync Releng templates for tyk-identity-broker:master

:robot: Beep, boop, this PR has been generated by [automation](https://github.com/TykTechnologies/gromit/blob/master/cmd/policy.go) at 2023-03-15 09:08:59.395827 &#43;0000 UTC

This PR updates the gpac engineering specific code in the master branch of tyk-identity-broker, and has been generated because some gpac code was changed, and the corresponding templates were updated.

These changes should be limited to the `repo-policy/` directory and `.github/workflows/pac.yml` file. This PR should be merged manually once the terraform manifest is applied by the SysE team and all branch protection rules are satisfied.

For more information about gpac and release branches please go to [Gpac Release Branches](https://tyktech.atlassian.net/wiki/spaces/EN/pages/1907228677/Release+branches#%5CuD83D%5CuDCD8-Instructions)

If you think this was made by mistake, or if merging this might cause any instabilities, please bring it up on Slack or log a jira in the TD project.

### CODE REVIEW

**Review by alephnull — APPROVED** [review]

File: repo-policy/main.tf [hunk 1]
```diff
@@ -36,14 +36,14 @@ module "tyk-identity-broker" {
 { branch    = "master",
 	reviewers = "1",
 	convos    = "false",
-	required_tests = ["1.16"]},
+	required_tests = ["1.15"]},
```
Comment (alephnull) [inline repo-policy/main.tf:39]: Sure about this change?

Comment (ermirizio) [reply repo-policy/main.tf]: I have change it now 😄

**Comment by ermirizio** [comment]
Terraform changes has been applied: 
https://app.terraform.io/app/Tyk/repo-policy-tyk-identity-broker/runs/run-2LgRsYYP4zxM9Fo7

I'm merging this one

**Comment by ermirizio** [comment]
Behaviour is intended, 

> Note: The option to enable auto-merge is shown only on pull requests that cannot be merged immediately. For example, when a branch protection rule enforces "Require pull request reviews before merging" or "Require status checks to pass before merging" and these conditions are not yet met. For more information, see "[Managing a branch protection rule](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/defining-the-mergeability-of-pull-requests/managing-a-branch-protection-rule)."

Link: https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/incorporating-changes-from-a-pull-request/automatically-merging-a-pull-request

**Pull request merged**